source/features/follow-file-renames.tsx

```diff
@@ -67,7 +71,8 @@ features.add({
 	screenshot: 'https://user-images.githubusercontent.com/1402241/54799957-7306a280-4c9a-11e9-86de-b9764ed93397.png'
 }, {
 	include: [
-		pageDetect.isRepoCommitList
+		// Only run on history pages
+		() => pageDetect.isRepoCommitList() && parseRoute(location.pathname).filePath.length > 0
```

Comment:
This parseRoute is duplicate, it’s already part of the init

Reply:
What do you mean I should do a return false instead of the inside the include?

Reply:
Right

Reply:
Done